### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by tdrozdovsky**
# Description
```
    Increase test coverage:

internal/common/errormsg/errormsg_test.go
internal/common/mqtt/mqttconnection_test.go
internal/restinterface/externalhandler/senderresolver/senderresolver_test.go
internal/restinterface/restinterface_test.go
internal/restinterface/route/route_test.go
internal/restinterface/tls/tls_test.go
```

## Type of change

- [x] Code cleanup/refactoring

# How Has This Been Tested?
```
make test
```
or 
```
make test internal/common/errormsg
...
```

**Test Configuration**:
* OS type & version: Ubuntu 20.04)
* Hardware: x86-64
* Toolchain: Docker v17.6 and Go v1.16)
* Edge Orchestration Release: v1.1.9

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: internal/restinterface/externalhandler/senderresolver/senderresolver_test.go [hunk 1]
```diff
@@ -32,49 +34,176 @@ var (
...
+
+		// 	time.Sleep(time.Microsecond * 100)
+
+		// 	_, err := http.Get("http://0.0.0.0:56001")
+		// 	if err != nil {
+		// 		println("unexpected error: ", err.Error())
+		// 		t.Error("unexpected error: ", err.Error())
+		// 	}
+
+		// 	wait.Wait()
```
Comment (t25kim) [inline internal/restinterface/externalhandler/senderresolver/senderresolver_test.go:161]: Can we delete comment code?

**Review by t25kim — APPROVED** [review]
LGTM Thanks!

File: internal/controller/servicemgr/executor/executor_test.go [hunk 2]
```diff
@@ -1 +1,30 @@
 package executor
+
```
Comment (MoonkiHong) [inline internal/controller/servicemgr/executor/executor_test.go:2]: Seems that we need to put the copyright statement and license at the top of this file. Thought? 😁

**Review by MoonkiHong — COMMENTED** [review]
Please take a look. 🙏

**Review by MoonkiHong — APPROVED** [review]
+1

File: .github/workflows/test-suite.yml [hunk 3]
```diff
@@ -24,6 +24,12 @@ jobs:
           echo "$HOME/go/bin" >> $GITHUB_PATH
           sudo mkdir -p /var/edge-orchestration/mnedc
           echo -e 'server-ip: 192.168.0.125\nport: 3334' | sudo tee /var/edge-orchestration/mnedc/client-config.yaml
+          if [ ! -e /var/edge-orchestration/certs/ca-crt.pem ]; then \
+		        cd tools ; ./gen_ca_cert.sh ; cp ./cert/ca-crt.pem /var/edge-orchestration/certs ; cd ../ 
+	        fi
+          if [ ! -e /var/edge-orchestration/certs/hen-crt.pem ]; then \
+		        cd tools ; ./gen_hen_cert.sh 1.1.1.1 ; cp ./cert/1.1.1.1/hen-*.pem /var/edge-orchestration/certs ; cd ../ 
+	        fi
```
Comment (t25kim) [inline .github/workflows/test-suite.yml:32]: Please check the indentation.

**Review by suresh-lc — APPROVED** [review]
+1

**Pull request merged**